eclipse/tests/src/nl/tudelft/watchdog/logic/interval/IntervalPersisterSingleIntervalTest.java

```diff
@@ -11,26 +11,32 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class IntervalPersisterTestSingleInterval extends IntervalPersisterTestBase {
+public class IntervalPersisterSingleIntervalTest extends IntervalPersisterTestBase {
 
 	@BeforeClass
-	public static void setUpBeforeClass() {
+	public static void setup_before_class() {
 		databaseName = "BaseTest";
 		setUpSuperClass();
 	}
-	
+
 	private static IntervalBase interval;
 
 	@Test
-	public void test1WriteInterval() {
+	public void can_compare_after_writes() {
+		writeInterval();
+		compareIntervalAfterWrite();
+		compareIntervalAfterWriteDemonstratesCloseIsNotPersisted();
+	}
+
+	private void writeInterval() {
```

Comment:
Why are these no longer tests?

Reply:
Per the PR description, these tests were actually 1 big test. In other words, if a previous test would fail the subsequent tests would fail as well in a non-descriptive manner. This was based on the Runner that would sort the methods alphabetically. However, tests should be independent and not rely on other tests.

Instead, the tests should be grouped in 1 JUnit test (in this case `can_compare_after_writes`). For reading comprehension, I kept the original methods which are then called in the full test. Since these original methods are not supposed to be independent tests, they should not be annotated with `@Test`